project/src/main/webapp/emailParse.js

```diff
@@ -11,92 +11,87 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-var model;
-function getModel()
-{
-    var modelLoadReady = document.getElementById("modelLoad");
-    modelLoadReady.innerHTML = 'Loading Model...';
-    console.time("Model Load");
-    qna.load().then(loadedModel => {
-        model = loadedModel;
-        console.timeEnd("Model Load");
-        
-        modelLoadReady.innerHTML = 'Model Ready';
-    });
-    
-}
+/** Class to load ML Model and find the best answer using email bodies */
+class mlModelEmailParse {
```

Comment:
Class names should be UpperCamelCase.

https://google.github.io/styleguide/jsguide.html#naming-class-names